src/views/form.hbs

```diff
@@ -393,6 +402,67 @@
+      <input type="radio" value="DK" name="C.7.3.2"> N/A<br>
+
+      <h2>C.7.3.3 Are controls placed at a minimum distance of 500 mm (600 mm recommended) from any adjacent corner or wall and between 900 mm and 1 200 mm above floor level, preferably 1 100 mm?</h2>
+      <input type="radio" value="Yes" name="C.7.3.3"> Yes<br>
+      <input type="radio" value="No" id="C.7.3.3" name="C.7.3.3"> No<br>
+      <input type="radio" value="DK" name="C.7.3.3"> N/A<br>
+
+      <h2>C.7.3.4 Are there Braille or tactile figures complementing the information in the control panel buttons?</h2>
+      <input type="radio" value="Yes" name="C.7.3.4"> Yes<br>
+      <input type="radio" value="No" id="C.7.3.4" name="C.7.3.4"> No<br>
+      <input type="radio" value="DK" name="C.7.3.4"> N/A<br>
+
+      <h1>C.7.4 Wayfinding and Signing</h1>
+      <h2>C.7.4.1 Are there tactile surface indicators in place (attention pattern and guiding pattern)?</h2>
+      <input type="radio" value="Yes" name="C.7.4.1"> Yes<br>
+      <input type="radio" value="No" id="C.7.4.1" name="C.7.4.1"> No<br>
+      <input type="radio" value="DK" name="C.7.4.1"> N/A<br>
+
+      <h2>C.7.4.2 Is a wayfinding system provided (map/tactile map of the facility, routes indicators, audio guidance)?</h2>
+      <input type="radio" value="Yes" name="C.7.4.2"> Yes<br>
+      <input type="radio" value="No" id="C.7.4.2" name="C.7.4.2"> No<br>
+      <input type="radio" value="DK" name="C.7.4.2"> N/A<br>
+
+      <h2>C.7.4.3 Are there signs designating rooms and spaces (room number, room?</h2>
+      <input type="radio" value="Yes" name="C.7.4.3"> Yes<br>
+      <input type="radio" value="No" id="C.7.4.3" name="C.7.4.3"> No<br>
+      <input type="radio" value="DK" name="C.7.4.3"> N/A<br>
+
+      <h2>C.7.4.4 The characters of the signs contrast with the background?</h2>
+      <input type="radio" value="Yes" name="C.7.4.4"> Yes<br>
+      <input type="radio" value="No" id="C.7.4.4" name="C.7.4.4"> No<br>
+      <input type="radio" value="DK" name="C.7.4.4"> N/A<br>
+
+      <h2>C.7.4.5 Are the text character raised?</h2>
+      <input type="radio" value="Yes" name="C.7.4.5"> Yes<br>
+      <input type="radio" value="No" id="C.7.4.5" name="C.7.4.5"> No<br>
+      <input type="radio" value="DK" name="C.7.4.5"> N/A<br>
+
+      <h2>C.7.4.6 Are signs mounted on the wall on the latch side of the door?</h2>
+      <input type="radio" value="Yes" name="C.7.4.6"> Yes<br>
+      <input type="radio" value="No" id="C.7.4.6" name="C.7.4.6"> No<br>
+      <input type="radio" value="DK" name="C.7.4.6"> N/A<br>
+
+      <h2>C.7.4.7 Are signs mounted between 120cm and max 160cm from the floor?</h2>
+      <input type="radio" value="Yes" name="C.7.4.7"> Yes<br>
+      <input type="radio" value="No" id="C.7.4.7" name="C.7.4.7"> No<br>
+      <input type="radio" value="DK" name="C.7.4.7"> N/A<br>
+
+
+      Comments: <textarea id="sectionCTextarea" rows="8" cols="80"></textarea><br>
+
+
```

Comment:
Replace all `<br>` elements with real css coding!
maybe
```css
display: flex;
flex-direction: column
```
this could be the style for your form container.

Reply:
@InassTubail